thirdeye-spi/src/main/java/ai/startree/thirdeye/spi/util/MetricsUtils.java

```diff
@@ -37,4 +42,8 @@ public static <E> E record(Callable<E> fun, final Timer successTimer, final Time
       throw e;
     }
   }
+
+  public static @NonNull String getNamespaceTagValue(@Nullable String namespace) {
```

Comment:
nit: 
this helper may be re-used a lot
for a static method of this kind, I tend to use
`namespaceTagValueOf`
instead of `get...`, because:
- it's a kind of conversion 
- `get...` tends to be used for many things like getters, building "important" objects and fetching stuff.

really just a nitpick